From c28a727a390e9e8656d2bd60df350acde29888fe Mon Sep 17 00:00:00 2001
From: Xiaoyu Yao <xyao@apache.org>
Date: Fri, 5 Feb 2016 10:37:07 -0800
Subject: [PATCH 1296/2748] HADOOP-12771. Fix typo in
 JvmPauseMonitor#getNumGcWarnThreadholdExceeded.
 Contributed by Xiaobing Zhou.

(cherry picked from commit 5d00067ca7f1460bbda1330e7329205a7b1ce019)
(cherry picked from commit e0a356fee3a885a2d61ced82bfd8d0969ccd05fa)

Change-Id: I888cbf7ebda1c6c1a2136de09e8e9bcdacf0123c
Conflicts:
---
 .../apache/hadoop/metrics2/source/JvmMetrics.java  |    2 +-
 .../org/apache/hadoop/util/JvmPauseMonitor.java    |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/source/JvmMetrics.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/source/JvmMetrics.java
index 164aa93..a6de50b 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/source/JvmMetrics.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/source/JvmMetrics.java
@@ -140,7 +140,7 @@ private void getGcUsage(MetricsRecordBuilder rb) {
     
     if (pauseMonitor != null) {
       rb.addCounter(GcNumWarnThresholdExceeded,
-          pauseMonitor.getNumGcWarnThreadholdExceeded());
+          pauseMonitor.getNumGcWarnThresholdExceeded());
       rb.addCounter(GcNumInfoThresholdExceeded,
           pauseMonitor.getNumGcInfoThresholdExceeded());
       rb.addCounter(GcTotalExtraSleepTime,
diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/JvmPauseMonitor.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/JvmPauseMonitor.java
index 17a1952..80d4468 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/JvmPauseMonitor.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/JvmPauseMonitor.java
@@ -106,7 +106,7 @@ public boolean isStarted() {
     return monitorThread != null;
   }
 
-  public long getNumGcWarnThreadholdExceeded() {
+  public long getNumGcWarnThresholdExceeded() {
     return numGcWarnThresholdExceeded;
   }
   
-- 
1.7.9.5

